django_x509/base/models.py

```diff
@@ -123,10 +123,11 @@ class BaseX509(models.Model):
                            help_text=_('additional x509 certificate extensions'),
                            load_kwargs={'object_pairs_hook': collections.OrderedDict},
                            dump_kwargs={'indent': 4})
-    serial_number = models.PositiveIntegerField(_('serial number'),
```

Comment:
remember to add a comment on why we have to use the `CharField` instead of the `PositiveIntegerField`